### PR TITLE
Add First Pass Candidate Detail

### PR DESCRIPTION
**Pull request by cgalbiati**
This feature is still in progress, but I thought you might want to take a look

### CODE REVIEW

File: src/screen-candidate-detail/viewSelectors.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { getCandidate } from '../candidate/redux/candidates';
+
+export const getCandidateSummary = (state, candidateId) => {
```
Comment (gmferland) [inline src/screen-candidate-detail/viewSelectors.js]: reselect is a good library for creating complex selectors that help bridge the gap between what we store in state and what we display in a view

**Review by gmferland — APPROVED** [review]

File: src/screen-candidate-detail/viewSelectors.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  const candidate = getCandidate(state, candidateId)
+  return {
+    name: 'Barach Obama',
```
Comment (crieggalder) [inline src/screen-candidate-detail/viewSelectors.js]: Barach, the lesser known Obama ;)

**Comment by gmferland** [comment]
I reconciled some merge conflicts and put the candidate detail screen in the appropriate place within the navigation structure of the app. However, after doing that I have some more stylistic questions that I wanted to discuss with the group

File: src/candidate/CandidateDetail.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, Text, View } from 'react-native';
+import { Avatar, Icon } from 'react-native-elements';
+
+const CandidateDetail = props => (
```
Comment (gmferland) [inline src/candidate/CandidateDetail.js:6]: I don't see the need for a wrapper component like this. As far as I can tell, its only function is to destructure the summary prop

File: src/candidate/CandidateDetailContainer.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import CandidateDetail from './CandidateDetail';
+
+const Container = connect(
+  (state, ownProps) => ({
+    summary: getCandidateSummary(state, ownProps.candidateId),
+  }),
+  { toggleFavorite },
+)(CandidateDetail);
+
+export default Container;
```
Comment (gmferland) [inline src/candidate/CandidateDetailContainer.js:13]: My personal style is to place the redux connected component in the same file alongside the plain react component. React-redux provides a property called `BaseComponent` that makes it very easy to access the plain component in unit tests or other cases where you don't want redux

**Review by mialknowles — APPROVED** [review]

**Pull request merged**